### PR TITLE
[G2M] Dev journal - update with asana automations and new journal workflow

### PR DESCRIPTION
**Pull request by hyx131**

### CODE REVIEW

File: communications/journal.md [hunk 1]
```diff
@@ -4,30 +4,28 @@ Journals don't always need to be like a ["Dear Diary" moment](https://gist.githu
...
-
-This also allows others in the team to know what you have been doing, so that more efficient collaboration can be formed from this knowledge.
+Our [avail_bot](https://github.com/EQWorks/avail-bot) routine for Dev Journal auto-fills _Last Workday_ with completed prev-day tasks, you could review & re-format these notes by directly typing into the Last Workday section.
 
 Adaptations:
 
-- Sessions can be in different lengths, such as every day, every other day, or every week.
-- However, it is better to have consistent session lengths. This allows members in collaboration with you to have a predictable period to check-in with you.
+- We have an automation tool to bring completed prev-day tasks to auto-fill this section.
+- Please edit this section if you find any missing prev-day tasks, or if prev-day tasks were poorly worded.
```
Comment (tlowande) [inline communications/journal.md:22]: seems redundant with the previous paragraph, I'd merge both and remove `adaptations`

File: workspace-automation.md [hunk 2]
```diff
@@ -33,3 +33,21 @@ This commands taps into our AWS Data Pipeline to see its current capacity.
 
 How it looks like in Slack:
 ![/pipeline](https://user-images.githubusercontent.com/2837532/72271944-cf55fc80-35f5-11ea-971c-e75a55148e67.png)
+
+## Asana
```
Comment (tlowande) [inline workspace-automation.md:37]: I would move this whole section to the beginning, prior to the legion slash commands

File: communications/journal.md (hunk 1 above)
Comment (woozyking) [reply communications/journal.md:22]: agree. the point-by-point presentation may sometime grab reading attention better. so perhaps adapt the previous paragraph into the points here, but revise the term and keep it consistent as _last workday_

File: workspace-automation.md (hunk 2 above)
Comment (woozyking) [reply workspace-automation.md:37]: oh this reminds me to write the github based updates and git based release automation docs too (the latter i may just link to the blog post lel)

Comment (hyx131) [reply workspace-automation.md]: kk @tlowande @woozyking i revised a bit, added some new screen shots as well as adding in annotations on the screen shots...lemme know if the annotations are too overwhelming 🌚

File: communications/journal.md [hunk 3]
```diff
@@ -4,56 +4,49 @@ Journals don't always need to be like a ["Dear Diary" moment](https://gist.githu
...
 
-Adaptations:
+Our [avail_bot](https://github.com/EQWorks/avail-bot) routine for Dev Journal auto-fills _Last Workday_ with completed prev-day tasks, please edit this section if you find any missing prev-day tasks, or if prev-day tasks were poorly worded. You could revise these notes by directly typing into the Last Workday section.
 
-- Sessions can be in different lengths, such as every day, every other day, or every week.
-- However, it is better to have consistent session lengths. This allows members in collaboration with you to have a predictable period to check-in with you.
+### (2) State Current Plan
 
-### Second Principle - State Current Plan
+State current plan in to-do tasks format, which could be mark completed as you progress:
```
Comment (woozyking) [inline communications/journal.md:25]: Maybe state as Subtasks?

File: communications/journal.md [hunk 4]
```diff
@@ -4,56 +4,49 @@ Journals don't always need to be like a ["Dear Diary" moment](https://gist.githu
...
+_Last Workday_ is a custom field in the journal:
 
-We can be much better at our planning by reinforcing and associating back to what we have done.
+<img width="903" alt="Screen Shot 2021-01-08 at 4 57 20 PM" src="https://user-images.githubusercontent.com/50936670/104070132-24f6c200-51d4-11eb-9ff1-177fec31b192.png">
 
-This also allows others in the team to know what you have been doing, so that more efficient collaboration can be formed from this knowledge.
+Notes in this section allows others in the team to know what you have been doing, so that more efficient collaboration can be formed from this knowledge. This section is also being sourced into weekly digests we generate for meeting notes, so it is key to keep them clear and concise.
 
-Adaptations:
+Our [avail_bot](https://github.com/EQWorks/avail-bot) routine for Dev Journal auto-fills _Last Workday_ with completed prev-day tasks, please edit this section if you find any missing prev-day tasks, or if prev-day tasks were poorly worded. You could revise these notes by directly typing into the Last Workday section.
```
Comment (woozyking) [inline communications/journal.md:21]: `avail-bot` for consistency

File: workspace-automation.md [hunk 5]
```diff
@@ -2,6 +2,24 @@
...
+
+### Dev Journal
+
+Asana Dev Journal project is listed under our Asana Dev team section. This project provides an interface for devs to use as a general journal entry where they would note down their tasks for the day, usually written in to-do format. 
+
+Our `avail-bot` runs a daily routine to create & assign dev-journal tasks for those who uses the journal. This automation formats completed prev-day subtasks into current day's `Last Workday` section, and incompleted prev-day subtasks will be re-created under the current day's subtasks section. This way, devs could could start their work by jumping right into left-over tasks from the previous day, and eliminates the tedious work of copy/paste prev-day taks to format current day's journal.
+
+Dev Journal is a project that helps to inform teammates of what each other is currently working on, providing a general scope of active projects among all teams. These daily updates will also be captured into weekly digests used to generate our meeting notes. Consult our [journal guidelines](https://github.com/EQWorks/common/blob/master/communications/journal.md) for more details.
```
Comment (woozyking) [inline workspace-automation.md:21]: github md has a pretty powerful dynamic in-repo linking mechanism:

```md
[journal guidelines](communications/journal.md)
```

like ^ would do and it'll link to the relevant file of the relevant branch

**Review by woozyking — APPROVED** [review]

**Review by tlowande — APPROVED** [review]

**Pull request merged**